### PR TITLE
Docs: k8s updates

### PR DESCRIPTION
**Pull request by becki-at-luminal**
This PR adds documentation for Kubernetes support.

### CODE REVIEW

File: docs/src/usage.md [hunk 1]
```diff
@@ -572,7 +576,7 @@ Files or directories passed in to `regula test` must include the following for e
 
 - The Rego rule file
 - The Rego tests file, where each test is prepended with `test_`
-- The test Terraform or CloudFormation IaC file
+- The test Terraform, CloudFormation, or Kubernetes file
```
Comment (ameliafugue) [inline docs/src/usage.md:579]: Should we say Kubernetes Manifest file instead of Kubernetes file?

**Review by ameliafugue — CHANGES_REQUESTED** [review]
I had one small comment, but other than that it lgtm

**Review by ameliafugue — APPROVED** [review]
LGTM 🎉

**Review by jason-fugue — APPROVED** [review]

**Review by curtis-fugue — APPROVED** [review]
Thank you!

**Pull request merged**